TestProjects/ShaderFoundry/Assets/CommonAssets/Editor/EditorTests/PropertyTests/BlockTestRenderer.cs

```diff
@@ -9,6 +9,12 @@ class BlockTestRenderer : FoundryTestRenderer
         internal static ShaderContainer CreateContainer()
         {
             var container = new ShaderContainer();
+
+            var builder = new ShaderType.StructBuilder(container, "VTPropertyWithTextureType");
+            builder.DeclaredExternally();
+            builder.AddInclude("Packages/com.unity.render-pipelines.core/ShaderLibrary/TextureStack.hlsl");
```

Comment:
Do we only support VT with SRP?

Reply:
As near as I know, yes. We at least currently need that file which is where all of the VT related stuff is defined.